src/components/Profile/ProfileLikes.tsx

```diff
@@ -3,33 +3,36 @@ import PostItem from "../Posts/PostItem";
 import axios from "axios";
 import { useAppDispatch, useAppSelector } from "../../state/hooks";
 import { Post, setPosts } from "../../state/slices/postsSlice";
+import { Divider, Stack } from "@mui/material";
+import { useParams } from "react-router-dom";
 
 const ProfileLikes = () => {
+  const { username } = useParams();
   const { posts } = useAppSelector((state) => state.posts);
-  const user = useAppSelector((state) => state.user);
   const dispatch = useAppDispatch();
 
   useEffect(() => {
     const fetchPosts = async () => {
       const result = await axios.get(
-        "http://localhost:3001/api/profile/getOwnLikes",
+        "http://localhost:3001/api/profile/getUserLikes",
         {
           params: {
-            userId: user.userId,
+            username,
```

Comment:
This makes me wonder if we should be using userId's or usernames for querying user info in the backend. It seems weird to use them interchangeably. I feel like we should be picking one and sticking with it for consistency

Now that I think about it I don't know why we have both a userId and a username, I think I just copied what I saw other people do. There must be a reason it's like that tho, maybe it's a performance thing? We should look into it and find out if it's better to query by username or by userId

I do see why we'd want the profile route to be /:username tho cause it makes it easier for people to type in a profile they're interested in. But maybe what ends up happening is we make the route /:username but query our API using the userId

Reply:
So I have updated the profile route to be using `/:username` however, I think if we want to make a decision for removing the use for `userId`, we should discuss later on and then finalize how we plan on using/not using them in another ticket